src/Dependencies/CodeAnalysis.Metadata/CustomDebugInfoEncoder.cs

```diff
@@ -194,7 +194,10 @@ public void AddTupleElementNames(IReadOnlyCollection<(string LocalName, int Slot
                         builder.WriteInt32(info.SlotIndex);
                         builder.WriteInt32(info.ScopeStart);
                         builder.WriteInt32(info.ScopeEnd);
-                        builder.WriteUTF8(info.LocalName);
+                        if (info.LocalName != null)
```

Comment:
❓ This code is new to me. Where would I look to see that it's fine to omit the value, as opposed to an approach like this:

```csharp
builder.WriteUTF8(info.LocalName ?? string.Empty);
```